src/components/TokenSideBar/TokenSideBarContent.test.tsx

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import { mount } from "enzyme";
+import { act } from "react-dom/test-utils";
+import TokenSideBarContent from "./TokenSideBarContent";
+
+describe("tokenSideBarContent", () => {
+  it("simulate input onchange events", () => {
+    // let mockInputValue1 = "";
+    // let mockInputValue2 = "";
+
+    const wrapper = mount(
+      <TokenSideBarContent
+        adminAddress="0xA"
+        holderAddress="0xA"
+        beneficiaryAddress="0xA"
+        approvedBeneficiaryAddress=""
+      />
+    );
+
+    const input1 = wrapper.find("#sec-transferholdership input");
+    const input2 = wrapper.find("#sec-changebeneficiary input");
+
+    // const inputMock1 = mount(input1.get(0));
+    // inputMock1.setProps({
+    //   value: mockInputValue1,
+    //   onChange: (event: any) => {
+    //     mockInputValue1 = event.target.value;
+    //   }
+    // });
+    // inputMock1.simulate("change", {
+    //   target: {
+    //     value: "Hello World 1"
+    //   }
+    // });
+
+    input1.simulate("change", {
+      target: {
+        value: "Hello World 1"
+      }
+    });
+
+    input2.simulate("change", {
+      target: {
+        value: "Hello World 2"
+      }
+    });
+
```

Comment:
simulate change is not testing anything. Can you test after change click on the button and it should call the service. You can mock the service response and check it after clicking its returning the same response.